jalejandromj/react-apprenticeship-capstone4
# Capstone-4_Deliverable-1

## Description
```
General Home page layout. Creation of Carousel and Slider components based on swiper and react-bootstrap (this last library was just used for slider functionality, no styles were used).
```
## Type of Change

```
Delete options that are not relevant.
```


## How should this be tested?
```
Add a brief description of how can be tested this deliverable step by step. 

- "npm run test" for testing + coverage
Home page should:
- Show playing slider at the top of the page
- Be able to drag n drop through "Shop our favourites" carousel
- Show a grid of 5 categories at the bottom of the page
```
## Any background context you want to provide?
```
N/A
```
## What is the deliverable?
```
Capstone 4. Deliverable 1
```
## Screenshots and links
```
https://alejandromj-wizecommerce.netlify.app/
```
## Notes
```
App is responsive for 1280, 1366, and 1920 until now.
```
## Checklist
